views/settings.html

```diff
@@ -178,10 +179,8 @@
     'premiumTabPremiumActiveMessage': __('MailPoet Premium is active', 'mailpoet'),
     'premiumTabPremiumInstallingMessage': __('MailPoet Premium plugin is being installed', 'mailpoet'),
     'premiumTabPremiumActivatingMessage': __('MailPoet Premium plugin is being activated', 'mailpoet'),
-    'premiumTabPremiumNotInstalledMessage': __('MailPoet Premium is not installed.', 'mailpoet'),
-    'premiumTabPremiumInstallMessage': __('Install MailPoet Premium plugin', 'mailpoet'),
-    'premiumTabPremiumNotActiveMessage': __('MailPoet Premium is not active.', 'mailpoet'),
-    'premiumTabPremiumActivateMessage': __('Activate MailPoet Premium plugin', 'mailpoet'),
+    'premiumTabPremiumNotActiveMessage': __('MailPoet Premium is not installed.', 'mailpoet'),
+    'premiumTabPremiumActivateMessage': __('Download MailPoet Premium plugin', 'mailpoet'),
```

Comment:
Can we use text stubs that fit a bit better? I find these two a bit confusing.
For example `premiumTabPremiumNotInstalledMessage` `premiumTabPremiumDownloadMessage `